install/linux/usr/share/odemis/sim/sparc2-ek-polarizer-sim.odm.yaml

```diff
@@ -0,0 +1,363 @@
+# Configuration for a simulated SPARCv2 system with polarization analyzer
+SPARC2: {
+    class: Microscope,
+    role: sparc2,
+    children: ["SEM E-beam", "SEM Detector", "Calibration Light",
+               "Optical Path Properties", "Spectrometer 1",
+               "Camera", "Spectrometer 2",
+               "Spec Filter Wheel", "Spec CCD Flipper", "Spectrograph focus",
+               "Mirror Actuators", "Mirror Actuators in XY referential",
+               "Slit", "Lens1 Mover", "Lens2 Switch",
+               "Polarization Analyzer", "Linear Polarizer", "Quarter Wave Plate"],
+}
+
+# Light (lamp with known spectrum)
+"Calibration Light": {
+    class: light.Light,
+    role: "brightlight",
+    power_supplier: "Power Control Unit",
+    affects: ["Camera", "Spectrometer 1", "Spectrometer 2"],
+}
+
+"Power Control Unit": {
+    class: powerctrl.PowerControlUnit,
+    role: "power-control",
+    init: {
+        port: "/dev/fake", # for simulator
+        pin_map: {"Calibration Light": 0, #"CL PMT control unit": 3,
+                  "Camera": 4, "Spectral Camera": 5, "Spectrograph": 6,
+                  "Optical Actuators": 2},
+        delay: { # Time it takes before a component is accessible
+            "Calibration Light": 0, # To turn on/off the light, it's immediate
+            "Camera": 3,
+            "Spectral Camera": 3,
+            "Spectrograph": 5,
+            "Optical Actuators": 1
+        },
+        init: { # Which component to power on from the very beginning (to save time)
+            "Camera": True,
+            "Spectral Camera": True,
+            "Spectrograph": True,
+            "Optical Actuators": True
+        },
+    }
+}
+
+# Can simulate a SEM connected to a DAQ board by using the comedi_test driver:
+# sudo modprobe comedi comedi_num_legacy_minors=4
+# sudo modprobe comedi_test
+# sudo comedi_config /dev/comedi0 comedi_test 1000000,1000000
+"SEM Scan Interface": {
+    class: semcomedi.SEMComedi,
+    role: null,
+    init: {device: "/dev/comedi0"},
+    # more detectors can be added, if necessary
+    children: {
+       scanner: "SEM E-beam",
+       detector0: "SEM Detector",
+    }
+}
```

Comment:
This could have been an !inclue isn't it? We use it in multiple microscope files so that would save quite some repetition between microscope files maybe.

Reply:
I'd rather have a self-contained file for the simulators. In any case, let's just keep this PR "simple" and not start reorganizing the microscope files now!